sway-core/tests/sway_to_ir/simple_contract_call.ir

```diff
@@ -7,62 +7,63 @@ script {
         local ptr { u64, b256 } s
 
         entry:
-        v0 = const { b256, u64, u64 } { b256 undef, u64 undef, u64 undef }, !1
-        v1 = const b256 0x0c1c50c2bf5ba4bb351b4249a2f5e7d86556fcb4a6ae90465ff6c86126eeb3c0, !2
-        v2 = insert_value v0, { b256, u64, u64 }, v1, 0, !1
-        v3 = const u64 2559618804, !1
-        v4 = insert_value v2, { b256, u64, u64 }, v3, 1, !1
```

Comment:
Unfortunately, the nature of these basic `diff` tests means you make a little change at the top and it cascades through for the rest.  I double checked and the only real difference is the addition of the `bitcast` instructions.